### PR TITLE
DT-13 add simple Profiles Controller and Profile Entity

### PR DESCRIPTION
**Pull request by andrejus-kuznecovas**

### CODE REVIEW

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BackendApartmentReservation.Controllers
+{
+    [Route("api")]
+    [ApiController]
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs:10]: Is `[ApiController]` the same as inheriting from `ControllerBase`?

File: BackendApartmentReservation/BackendApartmentReservation/Entities/Profile.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BackendApartmentReservation.Entities
+{
+    public class Profile
+    {
```
Comment (tomasmuzas) [inline BackendApartmentReservation/BackendApartmentReservation/Entities/Profile.cs:9]: Is `string` temporary until we have an Office entity?

File: BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs (hunk 1 above)
Comment (mstruckus) [reply BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs:10]: Pretty sure `ControllerBase` should be inherited from

Comment (andrejus-kuznecovas) [reply BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs:10]: Will add, thanks 😄

File: BackendApartmentReservation/BackendApartmentReservation/Entities/Profile.cs (hunk 2 above)
Comment (andrejus-kuznecovas) [reply BackendApartmentReservation/BackendApartmentReservation/Entities/Profile.cs:9]: Yes, will be changed to Office when it's implemented.  😄

**Review by mstruckus — APPROVED** [review]

**Review by tomasmuzas — APPROVED** [review]

**Pull request merged**